public/locales/es/TexterTodo.json

```
@@ -0,0 +1,7 @@
+{
+  "all gone": "¡Todo se ha ido!",
+  "error releasing": "Error al publicar respuestas:",
+  "done for the day": "¿Terminaste por hoy?",
+  "we can reassign": "Podemos reasignar conversaciones que no has respondido a otros mensajes de texto disponibles.",
```

Suggestion: I think this might be a verb/noun mixup
```suggestion
  "we can reassign": "Podemos reasignar conversaciones que no has respondido a otros mensajeros de texto disponibles.",
```